--- ./zenphoto/zp-core/plugins/admin_toolbox.php.orig	2008-12-13 20:00:12.907687267 +0100
+++ ./zenphoto/zp-core/plugins/admin_toolbox.php	2008-12-13 20:09:00.080910011 +0100
@@ -157,6 +157,18 @@
 		// close the list
 		echo "</ul>\n";
 		echo "</div>\n";
-	}
+        } else {
+                ?>
+                <script type="text/javascript">
+                var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
+                document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
+                </script>
+                        <script type="text/javascript">
+                try {
+                        var pageTracker = _gat._getTracker("UA-XXXXXX-Y");
+                        pageTracker._trackPageview();
+                } catch(err) {}</script>
+                <?php
+        }
 }
-?>
\ Pas de fin de ligne à la fin du fichier.
+?>
